### PR TITLE
Add layer information to the results

### PR DESCRIPTION
**Pull request by bcrochet**
Get some of the layer ids and other information from Crane and pass
it on to Pyxis results.

Signed-off-by: Brad P. Crochet <brad@redhat.com>

### CODE REVIEW

File: certification/internal/engine/engine.go [hunk 1]
```diff
@@ -342,22 +357,23 @@ func writeCertImage(img v1.Image) error {
...
+			Size:                   size,
+			UncompressedLayerSizes: layerSizes,
 		},
 		RawConfig:         string(rawConfig),
 		SumLayerSizeBytes: sumLayersSizeBytes,
-		// TODO: figure out how to get/convert this data from crane
-		UncompressedTopLayerId: "placeholder",
+		// This is an assumption that the DiffIDs are in order from base up.
+		// Need more evisdence that this is always the case.
+		UncompressedTopLayerId: config.RootFS.DiffIDs[0].String(),
```
Comment (acornett21) [inline certification/internal/engine/engine.go:376]: Non Blocking: If this is `top` level would this be the _last_ element in the array ie `config.RootFS.DiffIDs[len(config.RootFS.DiffIDs)-1]`?

**Review by acornett21 — APPROVED** [review]

File: certification/internal/engine/engine.go (hunk 1 above)
Comment (bcrochet) [reply certification/internal/engine/engine.go:376]: I thought so, but in my findings it's the first in the list.

Comment (acornett21) [reply certification/internal/engine/engine.go:376]: Okay, that's kind of confusing for sure. Happy to test more on this when the time comes.

**Pull request merged**